### DISCUSSION TITLE
Hacktribe does not read my SD Card

### DISCUSSION THREAD

**Discussion by jadeluxe** [bangcorrupt/hacktribe#193] [Q&A] [answered]
It's not possible for me to make it work. I tried to go back to the electribe 2 (blue) firmware but when I go to data utility/ software update it goes ERROR Not file. The same goes when I'm into data utility/card format. How can I go back to the factory reset? Thanks.

**Comment by bangcorrupt** [comment, marked as answer]
Have you edited the header of the factory firmware to look like the sampler firmware update file? #70

Are you running the latest version of Hacktribe?

Is the file at `KORG/hacktribe/System/SYSTEM.VSB`?

Can you export and import patterns to and from the SD card?

Do any of these help:

#149, #147, #138
